recipe/run_conda_forge_build_setup_win

```diff
@@ -9,6 +9,9 @@ conda install -n root --yes --quiet jinja2 anaconda-client
 # Workaround for Python 3.4 and x64 bug in latest conda-build.
 # FIXME: Remove once there is a release that fixes the upstream issue
 # ( https://github.com/conda/conda-build/issues/895 ).
-if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
+# if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
```

Comment:
@kmuehlbauer let's clean the comments as those above are no longer true.
Ideally we should remove this pin as soon as a new `conda-build` is out.